src/osd/modules/input/input_evdev.cpp

```diff
@@ -0,0 +1,713 @@
+// license:BSD-3-Clause
+// copyright-holders:Marc A. Pelletier
+//============================================================
+//
+//  input_evdev.cpp - Linux evdev input module
+//
+//  SDLMAME by Olivier Galibert and R. Belmont
+//
+//============================================================
+
+#include "input_module.h"
+#include "modules/osdmodule.h"
+
+#if defined(SDLMAME_SDL2) && (USE_EVDEV)
```

Comment:
Is this actually dependent on SDL at all (e.g. using SDL to get a window reference or something), or could it work with a different event loop?  It shouldn’t be checking for SDL if it doesn’t depend on it.

Reply:
(as above)

Reply:
It isn't, in any way.  In fact, that's the reason why I initially went with just USE_EVDEV.

I added that because I saw no obvious way to add options to Linux builds only beyond relying on SDL being the only Linux target right now, but that becomes entirely moot if we remove the options anyways.

So then the question becomes:  where do I _correctly_ add USE_EVDEV to the defines?  Directly as such in the top level makefile?